### PR TITLE
Remove Service Catalog references

### LINKED ISSUE #34138: 'Service Catalog' Helm repo URL is unavailable
[root@master opt]#helm repo add svc-cat https://kubernetes-sigs.github.io/service-catalog
Error: looks like "https://kubernetes-sigs.github.io/service-catalog" is not a valid chart repository or cannot be reached: failed to fetch https://kubernetes-sigs.github.io/service-catalog/index.yaml : 404 Not Found

this reporegistry URL from https://kubernetes.io/zh/docs/tasks/service-catalog/install-service-catalog-using-helm/.

### PR DESCRIPTION
**Pull request by Rishit-dagli**
As discussed in #34138 this PR removes all references to Service catalog

Fixes #34138

### CODE REVIEW

File: content/en/docs/reference/glossary/service-catalog.md
Comment (sftim) [inline content/en/docs/reference/glossary/service-catalog.md:17]: I think I'd leave the glossary entry in place, and reword it to make it clear that Service Catalog is a _former_ extension mechanism for Kubernetes.

Comment (Rishit-dagli) [reply content/en/docs/reference/glossary/service-catalog.md:17]: Ah, yes, that makes sense

**Comment by tengqm** [comment]
/lgtm

Defer approval to another reviewer, for I'd like this PR to be squashed.

**Comment by sftim** [comment]
/label tide/merge-method-squash

@Rishit-dagli would you be willing to squash this locally and force-push it?

**Review by sftim — DISMISSED** [review]
Another fix is needed.

It's OK for an approver to dismiss _this_ review once any change to the Service Catalog glossary page is made - and then, a new review is required.
I'll also aim to keep an eye on this PR. Nudge me if I miss it.

**Comment by Rishit-dagli** [comment]
> /label tide/merge-method-squash
> 
> @Rishit-dagli would you be willing to squash this locally and force-push it?

Done ✔️

File: content/en/docs/reference/glossary/service-catalog.md
Comment (sftim) [inline content/en/docs/reference/glossary/service-catalog.md:17]: Nearly there
```suggestion
It provided a way to list, provision, and bind with external {{< glossary_tooltip text="Managed Services" term_id="managed-service" >}} without needing detailed knowledge about how those services would be created or managed.
```
and then you can (should!) remove the glossary entry for Service Brokern.

**Review by sftim — COMMENTED** [review]
One more tweak and I think this will be good to go.

**Comment by jihoon-seo** [comment]
@Rishit-dagli
[Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/62b09cc343924f0008758d5f#L85).
> [ja] "docs/reference/glossary/service-catalog.md": "service-broker" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list

One possible solution might be:
- open a new PR that removes Service Catalog references in `content/ja/`
- and after that PR is merged, then rebase this PR with the latest `main`.

**Comment by Rishit-dagli** [comment]
> Merging https://github.com/kubernetes/website/pull/34439 should allow Netlify builds to work well.

I  have rebased this PR with the latest main, the Netlify previews now pass

**Comment by Rishit-dagli** [comment]
The following labels were automatically added due to a mistake I made in squashing the commits:
/remove-area blog
/remove-area release-eng
/remove-language ja
/remove-language pt
/remove-language ru
/remove-language zh
/remove-sig release

**Comment by jihoon-seo** [comment]
It seems that there remains no Service Catalog references after merging this PR.
/lgtm

**Pull request merged**
